core/src/main/resources/reference.conf

```diff
@@ -578,10 +578,8 @@ services {
     config {
       enabled = false
       auth.azure = false
-      # Notes:
-      #   - don't include the 'Bearer' before the token
-      #   - this config value should be removed when support for fetching tokens from ECM has been added to Cromwell
-      access-token = "dummy-token"
+      # Set this to the service that Cromwell should retrieve Github access token associated with user's token.
+      # ecm.base-url = "https://externalcreds.dsde-dev.broadinstitute.org/"
```

Comment:
My previous comment disappeared because of another change to this file (thanks Github) so just calling out that we talked about removing this URL in a now-outdated thread.

Reply:
Oops, I had the change locally but forgot to push it. `ecm.base-url` should now have value `https://example.org`